drivers/pinctrl/pinctrl_gecko.c

```diff
@@ -16,7 +16,13 @@ int pinctrl_configure_pins(const pinctrl_soc_pin_t *pins, uint8_t pin_cnt, uintp
 #ifdef CONFIG_SOC_FAMILY_SILABS_S1
 	LEUART_TypeDef *lebase = (LEUART_TypeDef *)reg;
 #else
+
+#if USART_COUNT > 1
 	int usart_num = USART_NUM(base);
+#else
+        int usart_num = 0;
+#endif /*USART_COUNT > 1*/
```

Comment:
I believe @asmellby prepare a PR to fix that.

Reply:
Hello, do you mean the PR: "drivers: spi: gecko: Apply frequency configuration, support devices with a single USART"? because I think those changes are only for the SPI driver

Reply:
No, I believe he didn't yet publish it. He is currently in vacation, I would like to have his opinion before to go further.

Reply:
Ok, thanks for your comment, I'll wait for him to review it.

Reply:
This is orthogonal to the changes I'm making, thanks for the fix!